netkan/netkan/cli.py

```diff
@@ -239,14 +237,14 @@ def clean_cache(days):
     '--debug', is_flag=True, default=False,
     help='Enable debug logging',
 )
-def download_counter(netkan, ckan_meta, token, key, debug):
+def download_counter(netkan_remote, ckan_meta, token, key, debug):
```

Comment:
Same as previous param doesn't match option.

Reply:
Fixed, now it works with either `--netkan` or `--netkan-remote`.